biopet-autodoc/templateparser.py

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env python2
+
+#Script to document new tools using a jinja2 template
+from jinja2 import Environment, FileSystemLoader
+import os
+import argparse
+import ast
+import yaml
+
+if __name__ == "__main__":
+    
+    #Block with command line options
+    parser=argparse.ArgumentParser(description="")
+    parser.add_argument("-t", type=str, nargs=1, help="Templatefile to be used")
```

Comment:
If you don't set `nargs`, accessing this value becomes easier, as you don't have to access a list.